index.html

```diff
@@ -203,21 +309,34 @@ <h2>About me</h2>
 
     <section id="contact">
       <form action="https://formspree.io/f/mqkonqbq" method="POST">
+        <h5>I'm always <br>interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+
         <ul id="form-ul">
-          <li>I'm always <br>interested in hearing about new projects, so if you'd like to chat please get in touch.</li>
-          <li>
+          <li id="li-full-name">
             <input placeholder="Fullname" name="fullname" 
             type="text" maxlength="30" required>
           </li>
-          <li>
+
+          <li id="li-first-name">
+            <input
+            placeholder="First-name" name="first-name" 
+            type="text" maxlength="30" required>
+          </li>
+
+          <li id="li-last-name">
```

Comment:
- [ ] kindly try more to stick to the figma design ,you need to work more on the form design as you can see according to the design must look exact like the figma template pleas kindly work more on the form user interface the textarea not visible loke the figma template
- [ ] your design
![you now](https://user-images.githubusercontent.com/92458236/236282914-59b0f49d-895b-466d-8c00-c7b07fd5b4a9.jpg)

- [ ] Figma
![fyyuu des](https://user-images.githubusercontent.com/92458236/236283014-c57e6154-2353-4a8c-83bb-fa2107ca1949.jpg)

Reply:
Yeah, it is hard to realize it, thank you @Adebowale

Reply:
You are welcome bro